lib/terminal_to_html.rb

```diff
@@ -9,7 +9,7 @@ def self.raw_render(raw)
   end
 
   def self.wrap_html_container(rendered)
-    stylesheet = File.read(File.join(Gem.latest_spec_for("terminal").full_gem_path, "/app/assets/stylesheets/terminal.css"))
+    stylesheet = File.read(File.join(Bundler.environment.specs["terminal"].first.full_gem_path, "/app/assets/stylesheets/terminal.css"))
```

Comment:
Once I wrote specs for the raw_stdout_html method, it turns out the original code was reaching out to rubygems.org, which was triggering a failure since webmock doesn't allow that.  This alternate implementation only looks locally at the gem on disk.

Reply:
Seems fine to me.  I might look into this later today to see if there is something in `Gem.*` that would work better and not make a web request, but I don't think it is worth pursuing now prior to a merge.

Reply:
If you can find something nicer, that would rock, but I couldn't find anything...we use this same Bundler line elsewhere in, I believe, Vmdb::Plugins, so this should be at least internally consistent.

Reply:
@Fryguy does that mean we're trying to talke to rubygems.org in prod also? I feel like that's a bigger issue than missing stdout on upgrades ...

Reply:
Yes, I believe it is talking to rubygems.org in prod also.  The new approach doesn't do that.

Reply:
Does that warrant making this a blocker do you think? As it stands this wouldn't get backported for GA I think.